src/elements/ExpiryIndicator.vue

```diff
@@ -0,0 +1,96 @@
+<script setup lang="ts">
+import ExpiryIcon from '@/icons/ExpiryIcon.vue';
+import {computed} from "vue";
+import { t } from '@/composable/i18n';
+
+// component properties
+interface Props {
+  days?: Integer;
+}
+const props = withDefaults(defineProps<Props>(), {
+  days: 10,
+})
+
+// Changes label from:
+// EXPIRES IN (n) DAYS > 5 days
+// EXPIRES IN 1 DAYS == 1 day
+// EXPIRED <= 0 days
+const days = computed(() => props.days);
+const label = computed(() => {
+  if (props.days <= 0) {
+    return "expired";
+  } else if (props.days == 1) {
+    return "expiresInDay";
+  } else {
+    return "expiresInDays";
+  }
+})
```

Comment:
Once the pluralization is in you can probably drop the label part, except for maybe translating enum -> string